src/app/pages/blog/article-summary/article-summary.tsx

```diff
@@ -2,35 +2,53 @@ import React from 'react';
 import RawHTML from '~/components/jsx-helpers/raw-html';
 import Byline from '~/components/byline/byline';
 import ClippedImage from '~/components/clipped-image/clipped-image';
+import type {
+    ArticleSummary as BlurbDataBase,
+    CollectionEntry,
+    SubjectEntry
+} from '~/pages/blog/blog-context';
 
-type Collection = {
-    name?: string;
-    value: {collection: Collection}[];
-};
+type CollectionVariant =
+    | CollectionEntry
+    | {
+          value: {collection: CollectionEntry}[];
+      };
+
+type ArticleSubjectVariant =
+    | SubjectEntry
+    | {
+          value: {subject: SubjectEntry}[];
+      };
+
+type BlurbData =
+    | null
+    | (Omit<BlurbDataBase, 'collections' | 'articleSubjects'> & {
+          meta?: {slug: string};
+          collections: CollectionVariant[];
+          articleSubjects: ArticleSubjectVariant[];
+      });
 
-type ArticleSubject = {
-    name?: string;
-    value: {subject: ArticleSubject}[];
-};
 
-type BlurbData = null | {
```

Comment:
BlurbData and ArticleSummaryData were defined independently, but BlurbData is really an extension of ArticleSummaryData.